src/shims/intrinsics/simd.rs

```diff
@@ -420,7 +420,6 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                     }
                 }
             }
-            #[rustfmt::skip]
```

Comment:
I wasn't sure about the purpose of this

Reply:
It avoided a bunch of reformatting that I considered undesirable. But now the branches have fairly different sizes anyway so it seems okay.